config/sources/families/include/crust_firmware.inc

```diff
@@ -19,17 +19,3 @@ declare -g CRUST_TARGET_MAP="scp;;build/scp/scp.bin"
 
 # Use a different BOOTDIR so that we don't leave scp.bin behind for non crust builds
 [[ -n "${CRUSTCONFIG}" ]] && BOOTDIR="u-boot-crust"
-
```

Comment:
Hey @viraniac -- this change inadvertently introduced a shortcircuit-as-last-statement bug here -- it used to define a function, which always succeeded, but now fails since the shortcircuit for BOOTDIR fails.